docs/ramalama.1.md

```diff
@@ -19,7 +19,7 @@ Running in containers eliminates the need for users to configure the host system
 
 RamaLama pulls AI Models from model registries. Starting a chatbot or a rest API service from a simple single command. Models are treated similarly to how Podman and Docker treat container images.
 
-When both Podman and Docker are installed, RamaLama defaults to Podman, The `RAMALAMA_CONTAINER_ENGINE=docker` environment variable can override this behavior. When neither are installed RamaLama attempts to run the model with software on the local system.
+When both Podman and Docker are installed, RamaLama defaults to Podman, The `RAMALAMA_CONTAINER_ENGINE=docker` environment variable can override this behaviour. When neither are installed RamaLama attempts to run the model with software on the local system.
```

Comment:
Both are correct depending on where you are.

Reply:
Yeah I've noticed this a few times, UK Vs US English. IMO just merge, because both are correct !